packages/mdc-switch/README.md

```diff
@@ -0,0 +1,49 @@
+# MDC Switch
+
+The MDC Switch component is a spec-aligned switch component adhering to the
+[Material Design Switch requirements](https://material.io/guidelines/components/selection-controls.html#selection-controls-switch).
+It works without JavaScript.
+
+## Installation
+
+```
+npm install --save @material/switch
+```
+
+## Usage
+
+### CSS Only
```

Comment:
I think we can just get rid of saying `CSS Only` here, since usually we say `CSS Only` when we want to specify how to use a component that requires Javascript, but gracefully degrades to a CSS Only implementation, e.g. textfields